workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/Program.cs

```diff
@@ -0,0 +1,315 @@
+
+                    Console.WriteLine(
+                        $"Creating the simulated player deployment {simDeploymentName} in project {projectName} with snapshot ID {simSnapshotId}.");
+                    var simDeploymentCreateOp = deploymentServiceClient.CreateDeployment(new CreateDeploymentRequest
+                    {
+                        Deployment = simDeploymentConfig
+                    }).PollUntilCompleted();
+                    Console.WriteLine("Successfully created the simulated player deployment.");
+                }
+            }
+            catch (Grpc.Core.RpcException e)
+            {
+                if (e.Status.StatusCode == Grpc.Core.StatusCode.NotFound)
+                {
+                    Console.WriteLine($"Unable to launch the deployment(s). This is likely because the project '{projectName}' or assembly '{assemblyName}' doesn't exist.");
+                }
+                else
+                {
+                    throw;
+                }
+            }
+
+            return 0;
+        }
+
+        private static int StopDeployment(string[] args)
+        {
+            var projectName = args[1];
+            var deploymentId = args[2];
+
+            var deploymentServiceClient = DeploymentServiceClient.Create();
+            try
+            {
+                deploymentServiceClient.StopDeployment(new StopDeploymentRequest
+                {
+                    Id = deploymentId,
+                    ProjectName = projectName
+                });
+            }
+            catch (Grpc.Core.RpcException e)
+            {
+                if (e.Status.StatusCode == Grpc.Core.StatusCode.NotFound)
+                {
+                    Console.WriteLine("<error:unknown-deployment>");
+                }
+                else
+                {
+                    throw;
+                }
+            }
+
+            return 0;
+        }
+
+        private static int ListDeployments(string[] args)
+        {
+            var projectName = args[1];
+
+            var deploymentServiceClient = DeploymentServiceClient.Create();
+            var listDeploymentsResult = deploymentServiceClient.ListDeployments(new ListDeploymentsRequest
+            {
+                ProjectName = projectName
+            });
+
+            foreach (var deployment in listDeploymentsResult)
+            {
+                if (deployment.Status == Deployment.Types.Status.Running)
+                {
+                    Console.WriteLine("<deployment> " + deployment.Id + " " + deployment.Name);
+                }
+            }
+
+            return 0;
+        }
+
+        private static void ShowUsage()
+        {
+            Console.WriteLine("Usage:");
+            Console.WriteLine("DeploymentManager.exe create <project-name> <assembly-name> <main-deployment-name> <main-deployment-json> <main-deployment-snapshot> [<sim-deployment-name> <sim-deployment-json> <sim-deployment-snapshot>]");
```

Comment:
I'd remove the .exe as this tool also runs on mac (or atleast, it should)

Reply:
Done!